sympy/solvers/ode/tests/ode_test_suite/test_kamke.py

```diff
@@ -0,0 +1,413 @@
+"""
+File to test Kamke ODEs. To run test suite, run
+
+    python test_kamke.py
+
+Information about failing ODEs is logged in test_report.txt.
+"""
+
+
+from contextlib import contextmanager
+import threading
+import _thread
+import time
+
+from sympy import (symbols, S, Function, Eq, dsolve, checkodesol,
+    Abs, sqrt, cos, exp, log, sin, tan, Derivative)
+
+
+class TimeOutError(Exception):
+    def __init__(self, msg=''):
+        self.msg = msg
+
+
+@contextmanager
+def time_limit(seconds, msg=''):
+    timer = threading.Timer(seconds, lambda: _thread.interrupt_main())
+    timer.start()
+    try:
+        yield
+    except KeyboardInterrupt:
+        raise TimeOutError("Timed out for operation {}".format(msg))
+    finally:
+        # Cancel timer if process finishes in time
+        timer.cancel()
+
+A, B, a, a0, a1, a2, a3, a4, alpha, b, b0, b1, b2, b3, b4, bbeta, c, d, ggamma, k, m, n, nu, x = symbols('A, B, a, a0, a1, a2, a3, a4, alpha, b, b0, b1, b2, b3, b4, bbeta, c, d, ggamma, k, m, n, nu, x')
```

Comment:
Why is it bbeta and ggamma instead of beta and gamma?

Reply:
I used `bbeta` and `ggamma` since `beta` and `gamma` might be used in the ODEs. That is not the case though, so I will rename them.